README.md

```diff
@@ -1 +1,5 @@
-# megafuel-js-sdk
\ No newline at end of file
+# megafuel-js-sdk
+
+This is Javascript SDK for the megafuel clients, it provides a simple way to interact with the megafuel.
```

Comment:
`megafuel` --> `MegaFuel`

Please have a guide about how to install MegaFuel js sdk,  and give a simple example about how to use the sdk to check sponsorable and send tx.

Reply:
fixed